keyboards/levinson/keymaps/jyh/keymap.c

```diff
@@ -0,0 +1,232 @@
+#include "levinson.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 3																/* Symbols, Media */
+#define _RAISE 4																/* Numbers, Arrows */
+#define _FUNC  5																/* Function Keys */
+#define _NUMS  6																/* Numpad */
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+// Fillers to make layering more clear
+#define KC_ KC_TRNS
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
These lines can be safely deleted as QMK defines these automatically in the core code (which is automatically included).

https://github.com/qmk/qmk_firmware/blob/27e216fc1a481bb3e65832458bd2b199cb499e0b/quantum/quantum_keycodes.h#L29-L31

```suggestion
```